test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -52,6 +52,13 @@ proc ::AlgorandGoal::Abort { ERROR } {
         puts "GLOBAL_TEST_ROOT_DIR $::GLOBAL_TEST_ROOT_DIR"
         puts "GLOBAL_NETWORK_NAME $::GLOBAL_NETWORK_NAME"
 
+        # dump system and processes into to check memory consumption and if algod procs are still alive
+        if { $tcl_platform(os) == "Darwin" } {
```

Comment:
Do these tests run on windows?

Reply:
I do not think so but added Linux case to be exact